src/backend/internal/model/model.go

```diff
@@ -32,6 +38,10 @@ type User struct {
 	Firstname          string    `bun:"firstname,type:varchar" json:"firstname"`
 	Lastname           string    `bun:"lastname,type:varchar" json:"lastname"`
 	VerificationStatus string    `bun:"verification_status,type:varchar" json:"verification_status"`
+	About              *string   `bun:"about,type:varchar" json:"about"`
+	Location           *string   `bun:"location,type:varchar" json:"location"`
```

Comment:
I believe this field is just the `address` of the user and does not have any logic tied with it -- serve only as a "to-read" information, does it?

If so, please consider changing it to `address`, to avoid confusion with the `location` field in the `package` entity, which will be used when a user adds location filter upon searching a package.